config.py

```diff
@@ -1,19 +1,22 @@
 # Discord API / Login settings
 TOKEN = 'put_token_here'
 
+game_stage = 0
```

Comment:
This should probably be stored on disk, in a file such as game_state.json along with other values such as season name, time, etc

Reply:
I was thinking of a dynamic config file, that'd be as easy to access as config.py, but was manageable and would change throughout the game.